### PR TITLE
Node op journey & snapshots

### PR DESCRIPTION
**Pull request by nsjames**

### CODE REVIEW

File: docs/40_node-operation/10_getting-started/50_snapshots.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+are stored in the `blocks.log` file. 
+
+## Syncing a fresh node
+
+To speed up the syncing process, you can download a snapshot and import it into your node. This will allow you to skip
+parts of the process that would otherwise take longer.
+
+### Required configuration
+
+You should have basic the following plugins enabled in your `config.ini` file:
```
Comment (iamveritas) [inline docs/40_node-operation/10_getting-started/50_snapshots.md:19]: the `basic` word seems out of place here.

File: docs/40_node-operation/10_getting-started/50_snapshots.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+
+## Creating a snapshot with full state history
+
+Creating snapshots allows you to create a backup of your node's state. This can be useful if you want to create periodic 
+backups of your node, or if you want to create a snapshot to share with others.
+
+### Required configuration
+
+In order to sync from a snapshot, you must have the `state_history_plugin` and `producer_api_plugin` plugins enabled.
```
Comment (iamveritas) [inline docs/40_node-operation/10_getting-started/50_snapshots.md:83]: wasn't this mentioned before in the `### Required configuration` section?

**Review by iamveritas — APPROVED** [review]
just a couple of comments left in there.

File: docs/40_node-operation/10_getting-started/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: Getting Started
+---
+
+<head>
+    <title>Getting Started with Node Operation</title>
+</head>
+
+This section digs into the basic knowledge you need to run a node. All node operators (apis, producers, history nodes)
+should know the information in this section.
```
Comment (lparisc) [inline docs/40_node-operation/10_getting-started/index.md:10]: should know -> should become familar with (or should get acquainted with)

(sounds more inviting)

File: docs/40_node-operation/10_getting-started/50_snapshots.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+If you are using a database filler, you need to look for `Placing initial state in block <block_num>` in the log. 
+
+Then you can start a filler with the following arguments:
+```shell
+... --fpg-create --fill-skip-to <block_num> --fill-trim
+```
+
+> On subsequent runs, you should not use the `--fpg-create` and `--fill-skip-to` arguments.
+
```
Comment (lparisc) [inline docs/40_node-operation/10_getting-started/50_snapshots.md:74]: Maybe add a short table of contents here for user to quickly select the procedure they want, as follows:

Select the procedure you need:
\- \[Create a snapshot with full state history](#creating-a-snapshot-with-full-state-history)
\- \[Restore a snapshot with full state history](#restoring-a-snapshot-with-full-state-history)
\- etc...

**Review by lparisc — APPROVED** [review]
Just a few suggestions.

File: docs/40_node-operation/10_getting-started/50_snapshots.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+### Using a database filler
+
+If you are using a database filler, you need to look for `Placing initial state in block <block_num>` in the log. 
+
+Then you can start a filler with the following arguments:
+```shell
+... --fpg-create --fill-skip-to <block_num> --fill-trim
+```
+
+> On subsequent runs, you should not use the `--fpg-create` and `--fill-skip-to` arguments.
```
[the remark this answers is not in the document]
Comment (nsjames) [reply docs/40_node-operation/10_getting-started/50_snapshots.md:73]: Actually just using it to highlight this, don't actually want an admonition. Guess I can just make it bold.

File: docs/40_node-operation/10_getting-started/50_snapshots.md (hunk 4 above)
Comment (nsjames) [reply docs/40_node-operation/10_getting-started/50_snapshots.md:74]: They have this already available on the sidebar in the UI, so we don't need to embed these in the docs anymore and bloat them

**Pull request merged**